src/Model/Tag.php

```diff
@@ -94,7 +94,7 @@ public static function store(int $uriid, int $type, string $name, string $url =
 		if (empty($cid)) {
 			$fields = ['name' => substr($name, 0, 96), 'url' => ''];
 
-			if (!empty($url) && ($url != $name)) {
+			if (($type != Tag::HASHTAG) && !empty($url) && ($url != $name)) {
 				$fields['url'] = strtolower($url);
```

Comment:
What is the point of this case lowering?

Reply:
We are storing the URL as varbinary. This means that the value is case sensitive. So we have to ensure on ourselfes that we don't create duplicates.

Reply:
I don't understand, why is the URL part of the unique index? Are looking terms up by URL? Otherwise, it would just be the same term, wouldn't it?

Reply:
The URL is used for mentions to someone we don't have the contact id. And you most likely do have two `Michael Vogel` as name - but with different URL.

Reply:
Ugh, thanks for the elaboration.